### DISCUSSION TITLE
Fix new Artifacts feature, which conflicts with ChatControls

### DISCUSSION THREAD

**Discussion by nordwestt** [open-webui/open-webui#6024] [Ideas]
Since it's my first time contributing this project, I've created this discussion. 

## Issue
When Artifacts is open, one cannot access the Controls.

[Video of issue.webm](https://github.com/user-attachments/assets/6d6a51c7-09ac-4d7a-af28-e0f011fb9ab0)

Furthermore, Artifacts does not seem to play well with small screen widths. 

## Changes:

- Fix artifacts being mixed up with Controls by refactoring into Chat component
- Hide Artifacts when width is too small 
- Create top-bar for Artifacts to prevent 'close' button from obstructing view of contents

Video of changes:
[Video of changes.webm](https://github.com/user-attachments/assets/a29d83e4-7386-4674-8893-af9f6711f0f5)

**Comment by chrisoutwright** [comment]
Is is possible to deactivate artifacts completely? In multi-chat I constantly get this opened whenever I switch conversation focus
